### ISSUE TITLE
Step missing in the installation guide: starting and enabling of the Docker daemon

### ISSUE THREAD

**Issue by houtepen** [nextcloud/all-in-one#4228]
### Is your feature request related to a problem? Please describe. 
An important step in the installation guide is missing (https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-use-this). The commands to start and enable docker are not included.

### Describe the solution you'd like

Step 3 should be something in the likes of: 
"Run the following commands to start and enable Docker:
systemctl start docker
systemctl enable docker"

### Describe alternatives you've considered

Starting the Docker daemon works.

### Additional context

The proposed solution above works. After that the command to start the container executes correctly instead of giving the error: "docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?"

**Comment by szaimen** [comment]
Hi, did you use the script as recommended and it did not automatically enable the systemd service?

**Comment by houtepen** [comment]
> Hi, did you use the script as recommended to install docker and it did not automatically enable the systemd service?

Hi, I followed step 1 as provided, so run "curl -fsSL https://get.docker.com | sudo sh". 

Step 3 did not work and gave the error "docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?". 

Before I was able to successfully perform step 3 I needed to run:
systemctl start docker
systemctl enable docker
